### PR TITLE
Initial commit of systemd-timesyncd

### PR DESCRIPTION
**Pull request by dasJ**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# systemd-timesyncd
+
+This role sets the timezone and configures and enables systemd-timesyncd.
+
+## Requirements
+
+systemd with timesyncd compiled in
```
Comment (SlothOfAnarchy) [inline README.md:7]: is this the case in Ubuntu/Arch?

File: templates/timesyncd.conf.j2 [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# vi: ft=dosini:
```
Comment (SlothOfAnarchy) [inline templates/timesyncd.conf.j2:1]: In what way is this line necessary for the role?

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dasJ) [reply README.md:7]: Yes.

File: templates/timesyncd.conf.j2 (hunk 2 above)
Comment (dasJ) [inline templates/timesyncd.conf.j2]: Yes.

Comment (dasJ) [reply templates/timesyncd.conf.j2:1]: It makes editing the role easier

Comment (SlothOfAnarchy) [reply templates/timesyncd.conf.j2:1]: What is there to edit, is your role not complete? I don't think optimizations for your local editor belong into the template file.

Comment (dasJ) [reply templates/timesyncd.conf.j2:1]: My role is complete, however there is the chance that the systemd developers change the interface or add new options. It is not just my local editor, it is the de-facto standard in the team everyone who is still active has agreed on, except for you.

**Review by SlothOfAnarchy — APPROVED** [review]

**Pull request merged**